packages/e2e-tests/test/e2e.spec.ts

```diff
@@ -886,6 +890,10 @@ describe('e2e', function () {
       it('interrupts load()', async function () {
         const filename = path.resolve(
           __dirname,
+          '..',
+          '..',
+          'cli-repl',
+          'test',
           'fixtures',
           'load',
           'infinite-loop.js'
```

Comment:
A bunch of these fixtures are only used in e2e tests, we can move those, right?

Reply:
My audit:

```
e2e:

load/hello1.js (both)
load/long-sleep.js
load/infinite-loop.js
load/throw.js (both)
exampleplayground.js
asi-script.js
require-base/
node-path/
curl.js

cli-repl:

fake-os-ca-provider.js
nodescript.js
load/a.js
load/b.js
load/c.js
load/hello1.js (both)
load/hello2.js
load/throw.js (both)
load/insertintotest.js
load/printisinteractive.js
```